src/indexables/application/cron-verification-gate.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Yoast\WP\SEO\Indexables\Application;
+
+use Yoast\WP\SEO\Helpers\Indexable_Helper;
+use Yoast\WP\SEO\Helpers\Options_Helper;
+
+/**
+ * The Cron_Verification_Gate class.
+ */
+class Cron_Verification_Gate {
```

Comment:
Yes in the perfect world I would make an infra class just to handle this part, but I found that in this case this could be in the gray area of we are a WordPress product, so a little WordPress here and there is accepted.